src/CodeGenerator/GeneratorTrait.php

```diff
@@ -80,4 +80,12 @@ public function setOutputDirectory(string $dir, ?int $mode = null) : self
 
         return $this;
     }
+
+    /**
+     * @return string
```

Comment:
Doc-block comment is useless here, as function has defined return type.
What's more it is not consistent with return type (or maybe it is on purpose?)

Reply:
It was auto generated by ZendStudio and i forgot to remove it ...